<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WaitingForm</class>
 <widget class="QWidget" name="WaitingForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="PokeImage1">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>131</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="PokeImage2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>40</y>
     <width>141</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="PokeInfo1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>160</y>
     <width>131</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="PokeInfo2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>160</y>
     <width>131</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>511</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Your Team</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
